news/4.33/platform.html

```diff
@@ -128,6 +128,21 @@ <h2>Text Editors </h2>
         </p>
       </td>
   </tr>
+  <tr id="sticky-scrolling"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1894 -->
+    <td class="title">Sticky Scrolling in Editors</td>
+    <td class="content">
+      <p>
+      Sticky Scrolling helps you keep track of your source code when working on large files by pinning the most important lines to the top of the source code editor.
+      It operates based on indentation, making it compatible with all different languages.
+      However, improperly formatted code may lead to suboptimal results which will be further improved in future releases.
+      </p>
+      <p><img style="max-width: 600px;" src="images/sticky-scrolling.png" alt="Sticky Scrolling"/></p>
+      <p>
+      You can enable Sticky Scrolling in the preferences under <b>General -> Editors -> Text Editors</b> by selecting the "Enable sticky scrolling" checkbox.
```

Comment:
Maybe you should phrase it similarly to how we have done it for the theme- ing topic:

        To try it out you can enable it in the preferences under <b>General -> Editors -> Text Editors</b> by selecting the <em>"Enable sticky scrolling"</em> checkbox. You can use 
        <a href="https://github.com/eclipse-platform/eclipse.platform.ui/issues/2152">issue #2152</a>
        to give your feedback. 
        </p>

Reply:
Yes, that's a good idea. Done.